internal/controllers/cryostat_controller.go

```diff
@@ -315,17 +317,18 @@ func (r *CryostatReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&operatorv1beta1.Cryostat{})
 
 	// Watch for changes to secondary resources and requeue the owner Cryostat
-	resources := []client.Object{&appsv1.Deployment{}, &corev1.Service{}, &corev1.Secret{}, &corev1.PersistentVolumeClaim{}}
+	resources := []client.Object{&appsv1.Deployment{}, &corev1.Service{}, &corev1.Secret{}, &corev1.PersistentVolumeClaim{},
+		&corev1.ServiceAccount{}, &rbacv1.Role{}, &rbacv1.RoleBinding{}, &netv1.Ingress{}}
```

Comment:
Should `ClusterRoleBinding` be included here too? Though, I am a bit unsure since our operator only works under single namespace now.

Reply:
Unfortunately it can't be because namespace-scoped objects (our CR) can't own cluster-scoped objects,

Reply:
> Also, I suppose these objects are watched only within the namespace of the operator?

Right, same as the CR itself. That will likely change in the future though.

Reply:
Right! That makes sense :D